lambda_functions/admin_service/template.yaml

```diff
@@ -0,0 +1,116 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: "SAM template for the DetektivKollektiv admin API"
+
+Globals:
+  Function:
+    Timeout: 30
+  Api:
+    OpenApiVersion: 3.0.1
+
+Parameters:
+  STAGE:
+    Type: String
+  CorsAllowOrigin:
+    Type: String
+  DBNAME:
+    Type: String
+    Default: development_db
+
+Resources:
+  Api:
+    Type: AWS::Serverless::Api
```

Comment:
Not entirely sure if it's possible, but if we could switch to `AWS::ApiGatewayV2::Api` instead of `AWS::Serverless::HttpApi` it would make things a lot easier. Especially in terms of CORS configuration.
If I understand it correctly, we would thereby make the special CORS method unnecessary. 

https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-httpapi.html

Reply:
Good point! However, I attached WAF rules to our api gateway instances and the HTTP Api doesn't seem to support this according to the documentation: https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-vs-rest.html
And now that we have already figured the CORS issues out for us, there would be no large benefit in making that change immediately, right? I would therefore suggest to keep this idea in mind and maybe wait until the HTTP API supports WAF and maybe make the change later.